### PR TITLE
ACM-10019 - 2. export helper functions

### PR DESCRIPTION
**Pull request by SherinV**
### Related Issue

https://issues.redhat.com/browse/ACM-10019

### Description of changes
- export helper functions

### CODE REVIEW

File: pkg/resolver/related.go [hunk 1]
```diff
@@ -388,15 +388,6 @@ func (s *SearchResult) updResultToCurrSearchUidsMap(resultUid string, currSearch
...
 
-func checkIfInArray(lookupMap []string, uid string) bool {
-	for _, id := range lookupMap {
-		if id == uid {
-			return true
-		}
-	}
-	return false
-}
-
```
Comment (SherinV) [inline pkg/resolver/related.go:399]: Moved to searchHelper.go file.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

File: pkg/resolver/searchHelper.go [hunk 2]
```diff
@@ -350,6 +350,15 @@ func pointerToStringArray(pointerArray []*string) []string {
 	return values
 }
 
+func CheckIfInArray(lookupMap []string, uid string) bool {
```
[the remark this answers is not in the document]
Comment (SherinV) [reply pkg/resolver/searchHelper.go:353]: Will update in a later PR. Thanks.

**Pull request merged**